app/src/main/java/com/dcrandroid/activities/RestoreWalletActivity.kt

```diff
@@ -115,6 +115,18 @@ class RestoreWalletActivity : BaseActivity() {
                 return@setOnClickListener
             }
 
+            // Compare seed with existing wallets seed.
+            var walletID = 1L
+            try {
+                walletID = multiWallet!!.walletWithSeed(enteredSeeds)
+            } catch (e: Exception) {
+                e.printStackTrace()
+            }
```

Comment:
if there's an error while trying to confirm seed, let's log it and display to the user
op can be `val op = this@RestoreWalletActivity.javaClass.name + ".walletWithSeed"`
```suggestion
            } catch (e: Exception) {
                e.printStackTrace()
                withContext(Dispatchers.Main) {
                    Dcrlibwallet.logT(op, e.message)
                    Utils.showErrorDialog(this, op + ": " + e.message)
                }
            }
```

same for the xpub